atlasdb-client/src/main/java/com/palantir/atlasdb/schema/stream/StreamTables.java

```diff
@@ -32,9 +32,10 @@
     public static final String HASH_TABLE_SUFFIX = "_stream_hash_idx";
     public static final String INDEX_TABLE_SUFFIX = "_stream_idx";
 
+    // todo can we remove this or at least rename it? It only generates the schema as if you had accepted all defaults and callers using the returned schema for other things could unknowingly blow away their changes
     public static void generateSchema(Schema schema, final String shortPrefix, final String longPrefix, final ValueType idType) {
         schema.addTableDefinition(shortPrefix + METADATA_TABLE_SUFFIX, getStreamMetadataDefinition(longPrefix, idType, ExpirationStrategy.NEVER, false, false));
-        schema.addTableDefinition(shortPrefix + VALUE_TABLE_SUFFIX, getStreamValueDefinition(longPrefix, idType, ExpirationStrategy.NEVER, false, false));
+        schema.addTableDefinition(shortPrefix + VALUE_TABLE_SUFFIX, getStreamValueDefinition(longPrefix, idType, ExpirationStrategy.NEVER, false, false, false));
```

Comment:
I can do some of them now though, I think she'll forgive me for at least a few merge conflicts.